src/Umbraco.Core/Persistence/Repositories/UserRepository.cs

```diff
@@ -320,51 +320,50 @@ public IEnumerable<IUser> GetUsersAssignedToSection(string sectionAlias)
         /// </remarks>
         public IEnumerable<IUser> GetPagedResultsByQuery(IQuery<IUser> query, int pageIndex, int pageSize, out int totalRecords, Expression<Func<IUser, string>> orderBy)
         {
-            if (orderBy == null) throw new ArgumentNullException("orderBy");
-
-            var sql = new Sql();
-            sql.Select("*").From<UserDto>();
-
-            Sql resultQuery;
-            if (query != null)
-            {
-                var translator = new SqlTranslator<IUser>(sql, query);
-                resultQuery = translator.Translate();
-            }
-            else
-            {
-                resultQuery = sql;
-            }
-
-            //get the referenced column name
+            // get the referenced column name and find the corresp mapped column name
             var expressionMember = ExpressionHelper.GetMemberInfo(orderBy);
-            //now find the mapped column name
             var mapper = MappingResolver.Current.ResolveMapperByType(typeof(IUser));
             var mappedField = mapper.Map(expressionMember.Name);
+
+            if (orderBy == null)

```

Comment:
No but `mappedField` only exists one line before checking for null or whitespace, that one is fine.